src/components/theme.js

```diff
@@ -13,9 +12,16 @@ export const backButtonTheme = {
 export const cardButtonTheme = {
   fg: 'white',
   bg: '#468DAE',
-  border: '1px',
   width: '150px',
   padding: '1px',
   shadow:
     '0px 1px 3px 0px rgba(0,0,0,0.2), 0px 1px 1px 0px rgba(0,0,0,0.14), 0px 2px 1px -1px rgba(0,0,0,0.12)'
 };
+
+export const fullZoomTheme = {
```

Comment:
I would strongly suggest having on theme object. Repetition is never our friend. It makes maintenance and expansion _soooo much harder_